lib/spring/json.rb

```diff
@@ -46,8 +46,6 @@ def self.dump(data)
 
 # See https://github.com/kr/okjson for updates.
```

Comment:
The upstream repo is archived so I think it's no big deal to modify it.

Also I wonder why `Marshal` wasn't used in the first place.